docs/topics/validation-report.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/rules-development-guide/master.adoc
 
+:_content-type: REFERENCE
 [id="validation-report_{context}"]
```

Comment:
@stoobie @sbeskin-redhat Modules with titles that begin "About..." are generally concept modules. Although this one has some details, I don't think it's a reference module.